appdata/il2cpp-types.h

```diff
@@ -4588,24 +4588,23 @@ namespace app
 #pragma endregion
 
 #pragma region PlayerVoteArea
-    struct PlayerVoteArea__Fields
-    {
+    struct PlayerVoteArea__Fields {
         struct MonoBehaviour__Fields _;
-        void* _Parent_k__BackingField;
+        struct MeetingHud* _Parent_k__BackingField;
         uint8_t TargetPlayerId;
         struct GameObject* Buttons;
-        void* ConfirmButton;
-        void* CancelButton;
-        void* PlayerButton;
-        void* PlayerIcon;
-        void* Background;
-        void* MaskArea;
-        void* Flag;
-        void* Megaphone;
-        void* Overlay;
-        void* XMark;
-        void* GAIcon;
-        void* ThumbsDown;
+        struct UiElement* ConfirmButton;
```

Comment:
UiElement is not defined anywhere. Stick with void* if you don't want to include definitions and don't need to use the field.